librz/bin/format/elf/rz_bin_elf_is_executable.c

```diff
@@ -0,0 +1,12 @@
+// SPDX-FileCopyrightText: 2009 Nibble <nibble.ds@gmail.com>
+// SPDX-FileCopyrightText: 2009 pancake <pancake@nopcode.org>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include "elf.h"
+
+bool Elf_(rz_bin_elf_is_executable)(RZ_NONNULL ELFOBJ *bin) {
```

Comment:
Splitting off this function seems a bit too fine-grained if it is just this single small one.